test/server/server.test.ts

```diff
@@ -0,0 +1,181 @@
+import {FastifyInstance} from 'fastify';
+import {Server} from 'http';
+import {
+  Catalog,
+  Dataset,
+  IRI,
+  Organization,
+  SparqlDistribution,
+} from '@netwerk-digitaal-erfgoed/network-of-terms-catalog';
+import {server} from '../../src/server/server';
+import {gql} from 'mercurius-codegen';
+import {setup, teardown} from 'jest-dev-server';
+
+let httpServer: FastifyInstance<Server>;
+describe('Server', () => {
```

Comment:
When going through this file, do the test cases read well? Is it clear what is going on?

Reply:
I like it!

A small hesitation about the data structure on lines 16-70: it's not instantly clear to me what the meaning of each element is. For example:

```
    new IRI('https://example.com/datasets/timeout'),
    'Dataset the Second',
    [
      new Organization(
        new IRI('https://example.com/organizations/2'),
        'Organization the Second',
        'O2'
      ),
    ],
```

`Dataset the Second` is probably the name of the dataset and `O2` probably the alternate name of the organization. However, is there a way to make this clearer?

Reply:
I’m using the `Organization`, `Dataset` etc. classes from our `catalog` package. IntelliJ (and possibly other editors such as VSCode) displays the arguments, which improves readability:

<img width="459" alt="Screenshot 2021-08-03 at 16 20 49" src="https://user-images.githubusercontent.com/89267/128031905-03773b6f-54cd-4a0e-bd90-ec38450e58bf.png">

On the code level I see two possible improvements:

1. Use more realistic examples, e.g. `Koninklijke Bibliotheek`, `KB`.
2. Use named arguments, which in JS is only possible using object destructuring: `{name: 'Dataset the Second', alternateName: 'O2'}`.

Reply:
Applied 1.